webroot/admin/group-mgmt.php

```diff
@@ -59,37 +65,46 @@
 include $LOC_HEADER;
 ?>
 
-<h1>PI Management</h1>
+<h1>Group Management</h1>
 <hr>
 
 <input type="text" id="tableSearch" placeholder="Search...">
 
-<h5>Pending PI Requests</h5>
+<h5>Pending Group Requests</h5>
 <table class="searchable">
     <tr class="key">
-        <td>Name</td>
-        <td>Unity ID</td>
+        <td>Group Name</td>
+        <td>Group Type</td>
+        <td>Requestor</td>
+        <td>Requestor UID</td>
         <td>Mail</td>
         <td>Requested On</td>
```

Comment:
If the group type has a start and end time, they should be shown in the requests table here. The column value can be left blank for types that don't have it